treeherder/webapp/api/performance_data.py

```diff
@@ -858,7 +863,9 @@ def _get_stddev_pct(self, avg, stddev):
         @param stddev: standard deviation of the runs values
         @return: standard deviation as percentage of the average
         """
-        return round(self._get_percentage(stddev, avg) * 100) / 100
+        if stddev:
+            return round(self._get_percentage(stddev, avg) * 100) / 100
+        return None
```

Comment:
maybe 0 is better than None ?

Reply:
@kimberlythegeek @alexandru-io  If there's only one run then the stddev can't be calculated and the stddev value will be None -> we can't calculate the stddev percentage if we don't have a value for stddev. Here's the initial [source code](https://github.com/mozilla/treeherder/blob/6c2aa0e15f54b6a0e2b1de25bd2ffc8d01f17822/ui/perfherder/perf-helpers/helpers.js#L51) where it states that if you don't have at least two run values then the stdev is `undefined`. 

Let me know if I'm missing something.